src/plette/pipfiles.py

```diff
@@ -42,6 +41,11 @@ def validate(cls, data):
                 continue
             klass.validate(data[key])
 
+        package_categories = set(data.keys()) - set(PIPFILE_SECTIONS.keys())        
```

Comment:
this will blow up for the pipenv:
```
NON_CATEGORY_SECTIONS = {
    "pipenv",
    "requires",
    "scripts",
    "source",
}
```

Reply:
Actually requires, scripts and sources are accounted for.  Its just the pipenv key.

Reply:
I don't understand your comment. Can you please elaborate?

Reply:
Is there a pipenv section in the Pipfile?

Reply:
Yeah, you can say things like `allow_prereleases = true` and there are a few other options.

Reply:
Another one is https://pipenv.pypa.io/en/latest/advanced/#specifying-package-indexes
`install_search_all_sources = true` 
These are things you want project wide that you wouldn't want individual developers having different settings in .env

Reply:
I see. Sigh... these really belong here, not it `project.py` in pipenv.

Reply:
Good catch btw. I added another commit to handle this section.